src/sparse/impl/KokkosSparse_spiluk_numeric_impl.hpp

```diff
@@ -242,52 +242,56 @@ struct ILUKLvlSchedTP1NumericFunctor {
 
   KOKKOS_INLINE_FUNCTION
   void operator()(const member_type &team) const {
-    auto my_league = team.league_rank();  // map to rowid
-    auto rowid     = level_idx(my_league + lev_start);
-    auto my_team   = team.team_rank();
-
-    auto k1 = L_row_map(rowid);
-    auto k2 = L_row_map(rowid + 1);
+    nnz_lno_t my_team = static_cast<nnz_lno_t>(team.league_rank());
+    nnz_lno_t rowid =
+        static_cast<nnz_lno_t>(level_idx(my_team + lev_start));  // map to rowid
+    // nnz_lno_t my_thread = static_cast<nnz_lno_t>(team.team_rank());
```

Comment:
We could clean up these two commented lines

Reply:
Thanks, @lucbv . These have been removed.